sc/social/like/browser/viewlets.py

```diff
@@ -74,6 +89,49 @@ def enabled(self):
             return True
         return self.helper.enabled(self.view)
 
+    @property
+    def via(self):
+        record = ISocialLikeSettings.__identifier__ + '.twitter_username'
+        try:
+            return api.portal.get_registry_record(record)
```

Comment:
`api.portal.get_registry_record()` now supports a default value, use it instead.